### ISSUE TITLE
Reconsider Keep Mint Green

### ISSUE THREAD

**Issue by Gluzman** [keep-network/website#202]
Adjust the green for contrast issues.

**Comment by Gluzman** [comment]
![keep-argent](https://user-images.githubusercontent.com/7621557/85457967-9d66a000-b56e-11ea-90e9-eedcb37e5ad8.png)

**Comment by liz-shinn** [comment]
Contrast issues in text use of the mint color as well. Fails accessibility at large scale type and small type. Need another darker shade for use with text.

<img width="555" alt="Screen Shot 2020-10-09 at 4 35 27 PM" src="https://user-images.githubusercontent.com/57226633/95629286-98f6c200-0a4d-11eb-9b98-1ca93dcc5a70.png">

**Comment by liz-shinn** [comment]
Updated this – added two darker shades to use for color contrast issues with text.
<img width="610" alt="Screen Shot 2020-10-09 at 5 23 16 PM" src="https://user-images.githubusercontent.com/57226633/95632673-2f2de680-0a54-11eb-9e0c-42068ffd1187.png">

Darker mint color passes accessibility now, can use this color for text.
<img width="576" alt="Screen Shot 2020-10-09 at 5 24 00 PM" src="https://user-images.githubusercontent.com/57226633/95632692-3a811200-0a54-11eb-8f0c-6c179488890b.png">

**Comment by liz-shinn** [comment]
Added to Keep Design System:

<img width="563" alt="Screen Shot 2020-10-09 at 5 26 42 PM" src="https://user-images.githubusercontent.com/57226633/95632893-9b104f00-0a54-11eb-8b19-c495849b42da.png">
